result.js

```diff
@@ -20,16 +20,16 @@
 
 module.exports = Result;
 
-function Result(error, value) {
+function Result(err, value) {
```

Comment:
Wait, why? This is backward incompatible, causes churn downstream, and reduces sensibility of the interface?